src/main/kotlin/graphics/scenery/utils/pointers.kt

```diff
@@ -0,0 +1,31 @@
+package graphics.scenery.utils
+
+import kool.Ptr
+import kool.unsafe
```

Comment:
:exclamation: unsafe? Like `sun.misc.Unsafe`? Do you discourage its use? Or...?

Reply:
Yep, atm it's the way for low level native operations

The whole Lwjgl ultimately rely on that as well